vm-setup/roles/v1aX_integration_test/tasks/download_image.yml

```diff
@@ -6,9 +6,7 @@
 
   - debug:
       msg: "Local image of CentOS is found"
```

Comment:
This is not centos only anymore

Reply:
I think it still downloads only the CentOS, the only the difference is that there is no condition to run this playbook, no?

Reply:
Yes, it should be for all images